CONTRIBUTING.md

```diff
@@ -6,7 +6,7 @@
 * **Step 1.** Get project dependencies
 ```bash
 brew install go
-brew install glide
+brew install dep
```

Comment:
We collect our development dependencies into `node-builder` image here `bin/builder_docker/Dockerfile`